### ISSUE TITLE
stop pause with display

### ISSUE THREAD

**Issue by frederic2712** [Dazix/SimpleCountDown#6]
Hello,
I think is not possible to display the countup clock without starting the counter.
ex:00:00:00 
Then it seems not possible to pause, reset, and restart it during run.

**Comment by Dazix** [comment]
Yes it is true. This is because the time is counted from / to a certain date. So only animations can be stopped. So basically creating an instance is the beginning of counting.
